### ISSUE TITLE
TYPO3 12 compatibility

### ISSUE THREAD

**Issue by svewap** [dmind-gmbh/extension-cookieman#304]
Here is my version for T3 12: https://github.com/svewap/extension-cookieman/tree/12-svewap
I can make a pull request if you want.

**Comment by jonaseberle** [comment]
Thanks Sven!
Yes, please open a PR :) 
I have branched off `11lts` so we can work on v12 together. 

There is https://github.com/dmind-gmbh/extension-cookieman/issues/271 which needs to be solved.

I am planning to work next week on it.

**Comment by weisgerberm** [comment]
Hi, any Updates for v12 support? Thank you

**Comment by zenoussi** [comment]
> Hi, any Updates for v12 support? Thank you

Same question. Thanks

**Comment by svewap** [comment]
I opened a PR

**Comment by ingeniumdesign** [comment]
new Infos?

**Comment by jonaseberle** [comment]
@msollmann You mean in the T3 v12 branch?

**Comment by msollmann** [comment]
In https://github.com/dmind-gmbh/extension-cookieman/pull/311. 

In addition I can only override my own labels (e.g. "group.externalServices"), not existing labels (e.g. "group.marketing") via locallangXMLOverride.

**Comment by ingeniumdesign** [comment]
new Infos?

**Comment by Finulin** [comment]
release for TYPO3 v12 would be nice

**Comment by jonaseberle** [comment]
Yes, I agree, but I have not found the time. It would really help if you could test #311 and leave feedback there.

If someone would like to dive into the docs and adjust the examples where needed, that would help as well.

**Comment by jonaseberle** [comment]
@msollmann your translation problem is https://forge.typo3.org/issues/100759

**Comment by ingeniumdesign** [comment]
Maybe as a christmas present? :)

**Comment by jonaseberle** [comment]
@ingeniumdesign yes, thank you, please go ahead and test https://github.com/dmind-gmbh/extension-cookieman/pull/323 ;)
`composer req dmind/cookieman:dev-dev/12lts`.

**Comment by ingeniumdesign** [comment]
The installation worked fine, the output works fine, now I'll test the settings a bit more

**Comment by ingeniumdesign** [comment]
yes, the  _LOCAL_LANG TypoScript issue dont work. New Infos?

**Comment by jonaseberle** [comment]
@ingeniumdesign Are you adding the extensionName in translations like this? `{f:translate(key: 'heading', extensionName: 'cookieman')}`

**Comment by jankornblum** [comment]
For me everything works fine with 12 lts. When will you publish official version? I have many projects waiting to get upgraded to 12 lts, where cookieman is the only blocker :(

**Comment by jonaseberle** [comment]
I am sorry. The docs are still not ready https://github.com/dmind-gmbh/extension-cookieman/pull/323. 

You can pin the latest commit with `composer req dmind/cookieman:dev-dev/12lts#cc9364f4ba76de706f4af2a3e1be7372aaabba09`.

**Comment by jankornblum** [comment]
Thanks, yes, but I don't like to require a dev-branch in a live project so much ;)

**Comment by jonaseberle** [comment]
You would pin a commit that you have tested. But ok, I understand what you mean.

**Comment by plojewski** [comment]
When can we expect this version in the Typo3 repository?

**Comment by jonaseberle** [comment]
It would help if someone with documentation skills would add a PR tackling the open issues https://github.com/dmind-gmbh/extension-cookieman/pull/323

**Comment by ingeniumdesign** [comment]
New Infos?

**Comment by ingeniumdesign** [comment]
Wuhuuu 🚀🚀🥳🙏
